docs/data/toolpad/core/introduction/base-concepts.md

```diff
@@ -105,3 +105,124 @@ In this example:
 
 - The `slots` prop allows you to replace entire parts of the component.
 - The `slotProps` prop lets you pass additional props to specific slots.
+
+## Localization
+
+Toolpad components support translations between languages. You can modify text and translations inside Toolpad components in several ways.
+
+The default locale is English (United States). To use other locales, follow the instructions below.
+
+### Set translations globally
+
+#### Using the theme
+
+To translate all your Toolpad components, you can provide translations through the theme:
+
+```tsx
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import hiIN from '@toolpad/core/locales/hiIN';
+
+const theme = createTheme({
+   {
+    palette: {
+      primary: { main: '#1976d2' },
+    },
+  },
```

Comment:
Should we add core locale as well? That would be the user's intent, likely, rigth?